process/transaction/export_test.go

```diff
@@ -57,9 +58,10 @@ func (txProc *txProcessor) ProcessUserTx(
 
 func (txProc *txProcessor) TakeMoveBalanceCostOutOfUser(
```

Comment:
Rename also this method ProcessMoveBalanceCostRelayedUserTx

Reply:
done.